server/main.lua

```diff
@@ -44,4 +44,12 @@ function IsAuthorized(jobName, doorID)
 	end
 
 	return false
-end
\ No newline at end of file
+end
+
+RegisterServerEvent('esx_doorlock:registerDoors')
+AddEventHandler('esx_doorlock:registerDoors', function(doors)
+	for _, doorInfo in pairs(Config.DoorList) do
```

Comment:
isn't it `ipairs` also remove the spacing in `for _, doorInfo` to look like this `for _,doorInfo`